book/website/ethical-research/law-policy.md

```diff
@@ -0,0 +1,251 @@
+Each of these instruments is overseen by a committee of experts: a 'treaty body.'
+Every few years, each country who has signed a human rights instrument has to report to the treaty body on how well they are implementing the treaty.
+The treaty body makes recommendations on how the country can continue to improve its implementation.
+The treaty bodies also periodically issue more detailed guidance on how to interpret different articles of their treaty, to help countries understand better what their obligations.
+
+There are also regional human rights instruments:
+- [African Charter on Human and People's Rights](https://au.int/en/treaties/african-charter-human-and-peoples-rights), which covers 54 of the 55 members of the [African Union](https://au.int/en/member_states/countryprofiles2).
+- [American Convention on Human Rights](http://cidh.oas.org/Basicos/English/Basic3.American%20Convention.htm), which covers 23 of the 35 members of the [Organization of American States](http://www.oas.org/en/about/member_states.asp).
+- [European Convention on Human Rights](https://www.coe.int/en/web/conventions/full-list/-/conventions/treaty/005), which covers the 47 members of the [Council of Europe](https://www.coe.int/en/web/about-us/our-member-states).
+- [Charter of Fundamental Rights of the European Union](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX:12012P/TXT), which covers the 27 members of the [European Union](https://europa.eu/european-union/about-eu/countries_en)
+
+Some countries have ratified some of the treaties that apply to them: this means that they have incorporated the content of the the treaties into their national law.
+Many countries also have a national human rights institution, which independently monitors and investigates human rights violations in that country, and advises the government on implementing their obligations. 
+
+Countries that have signed treaties have committed to implementing them.
+However, the process of implementing treaties in full, so that every person in a country can enjoy all the rights in a treaty, is very long and requires lots of actions from states.
+Campaigning groups, including non-governmental organisations, work to narrow the gap between what countries have promised to do, and what they have actually done so far. 
+Some campaigning groups are international and work on lots of different human rights issues. 
+Some work only on a small number of issues, or in a small area of the world.
+
+### How do human rights apply to data science and research ethics?
+
+Human rights are more concrete than ethics, because they are defined in law. 
+There is a wider shared understanding of what human rights means across different countries, cultures and groups of people.
+There are accountability and enforcement mechanisms.
+But because they are a legal framework, they are more difficult to understand and apply to the daily work of data science and research.
```

Comment:
```suggestion
But because they are a legal framework, they are not as easily accessible, and hard to apply to the daily work of data science and research.
```